### ISSUE TITLE
Fronius-inverter node not shown

### ISSUE THREAD

**Issue by ahpohl** [mabunixda/node-red-contrib-fronius-solar#32]
After installing the `node-red-contrib-fronius-solar` palette I cannot see the `inverter` node. The `control in` node is there.

![image](https://github.com/mabunixda/node-red-contrib-fronius-solar/assets/30575522/19fd9b93-d150-4a38-bca4-5a45e0847a25)

The node-red admin list shows that the palette has been successfully installed.
```
Nodes                                    Types                State  
node-red-contrib-fronius-solar/fronius   fronius-control      enabled
                                         fronius-inverter            
```
Am I looking in the wrong place?

**Comment by mabunixda** [comment]
The inverter is the configuration item in the background to control the control node

**Comment by ahpohl** [comment]
Ok, now I entered the hostname and api version into the control node properties. But there is still no output. There is really not a lot of documentation how to use the fronius-solar module. Could you provide an example?

For sure, I don't see the fronius-inverter node like shown on the [node-red page](https://flows.nodered.org/node/node-red-contrib-fronius-solar):

![image](https://github.com/mabunixda/node-red-contrib-fronius-solar/assets/30575522/7fef99a6-37b1-423e-bbae-924014819971)

I have connected like this, which is obviously wrong. I don't want to control it, I just want to get the inverter output say every 5 seconds for further processing. For this I would need the fronius-inverter node I assume.

![image](https://github.com/mabunixda/node-red-contrib-fronius-solar/assets/30575522/3b6ec58e-d368-4d3f-a4bf-f438000296c8)

**Comment by mabunixda** [comment]
You need an inject node to connect as an input

**Comment by ahpohl** [comment]
Ok, now I connected an `inject` node to the input of the `fronius control in` node. But what kind of payload do I need now to get some output? I haven't found an example or documentation how to use it.

**Comment by mabunixda** [comment]
Yeah just define in inject node with some interval configuration like
![image](https://github.com/mabunixda/node-red-contrib-fronius-solar/assets/1193145/db5dd285-c98a-4ab7-8947-8ae15d154d84)
